G_channels.md

```diff
@@ -249,6 +249,24 @@ socket = assign(socket, :user, msg["user"])
 
 Sockets store assigned values as a map in `socket.assigns`.
 
+#### Failure Semantics
+
+Servers will restart & clients will fail - understanding how Phoenix responds to these events and the guarantees it offers will  help you design robust systems. 
```

Comment:
Little things:
- Please change the & to "and".
- There is an extra space in ". . . offers will  help . . . "
- Please change the "you" to "us" in ". . . help you design robust . . ." It's a subtle thing, but we've tried to adopt the more inclusive usage wherever possible in the guides.